src/factories/series/Column.ts

```diff
@@ -9,50 +9,66 @@ module n3Charts.Factory.Series {
     protected seriesClass: string = Column.type + '-series';
     protected dataClass: string = Column.type;
 
-    public colsWidth: number = 0;
+    private gapFactor: number = 0.2;
+    private outerPadding: number = (this.gapFactor / 2) * 3;
+    private columnsWidth: number = 0;
+
+    public x1Scale: D3.Scale.OrdinalScale;
 
     update(data: Utils.Data, options: Utils.Options) {
       super.update(data, options);
 
       var series = options.getSeriesForType(Column.type);
 
-      this.updateColsWidth(series);
+      this.updateColumnsWidth(series, options);
```

Comment:
In `SeriesFactory` we can access the options with `this.options`. Is there a good reason not to do so?